iib/workers/tasks/build.py

```diff
@@ -120,7 +122,11 @@ def _cleanup():
 
 
 @retry(exceptions=IIBError, tries=get_worker_config().iib_total_attempts, logger=log)
-def _create_and_push_manifest_list(request_id, arches, build_tags):
+def _create_and_push_manifest_list(
+    request_id: int,
+    arches: Iterable[str],
```

Comment:
Why are we using Iterable here and not List?

Reply:
That´s a good point, I used that as i saw it elsewhere already used for arches. 
Now I changed arches, and all add_arches function parameters to have type Set - as the arches should not repeat itself. 
If I am wrong, please let me know, I will change those to List type.

Reply:
Set is better :+1: